File: app/src/main/java/com/wang17/myclock/database/BankToDo.java
package com.wang17.myclock.database;

import com.wang17.myclock.model.DateTime;

import java.util.UUID;

public class BankToDo{

    private UUID id;
    private DateTime dateTime;
    private String bankName;
    private String cardNumber;
    private double money;

    public BankToDo(DateTime dateTime, String bankName, String cardNumber, double money) {
        this.id = UUID.randomUUID();
        this.dateTime = dateTime;
        this.bankName = bankName;
        this.cardNumber = cardNumber;
        this.money = money;
    }

    public DateTime getDateTime() {
        return dateTime;
    }

    public void setDateTime(DateTime dateTime) {
        this.dateTime = dateTime;
    }


    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getCardNumber() {
        return cardNumber;
    }

    public void setCardNumber(String cardNumber) {
        this.cardNumber = cardNumber;
    }

    public String getBankName() {
        return bankName;
    }

    public void setBankName(String bankName) {
        this.bankName = bankName;
    }

    public double getMoney() {
        return money;
    }

    public void setMoney(double money) {
        this.money = money;
    }
}

File: app/src/main/java/com/wang17/myclock/database/BankToDo.kt
package com.wang17.myclock.database

import com.wang17.myclock.model.DateTime
import java.util.*

class BankToDo {
    var id: UUID
    var dateTime: DateTime
    var bankName: String
    var cardNumber: String
    var money: Double

    init {
        id = UUID.randomUUID()
        dateTime = DateTime()
        bankName = ""
        cardNumber = ""
        money =0.0
    }

    constructor()

    constructor(id:UUID,dateTime: DateTime,bankName:String,cardNumber:String,money:Double){
        this.id = id
        this.dateTime = dateTime
        this.bankName = bankName
        this.cardNumber = cardNumber
        this.money =money
    }
}
